1.9/administration/metrics/metrics-api.md

```diff
@@ -3,14 +3,35 @@ post_title: Metrics API
 menu_order: 1
 ---
 
+You can use your custom built tools to get any metric, either from running Mesos tasks or the hosts which run your DC/OS cluster. 
+
+
+# About the Metrics API
+
+The HTTP producer is enabled by default and exposes a JSON-formatted HTTP API on each node in the cluster. These APIs include both metrics datapoints as well as dimensions, or key/value pairs with relevant node and cluster metadata.
+
+# Request and response format
+
+The API supports JSON only. You must include application/json as your Content-Type in the HTTP header, as shown below.
+
+```bash
+Content-Type: application/json
+```
```

Comment:
The API returns `Content-Type: application/json` but the user does not need to specify this content type, as this is a read-only API.